environments/skeleton/{{cookiecutter.environment}}/terraform/compute/nodes.tf

```diff
@@ -45,9 +45,16 @@ resource "openstack_compute_instance_v2" "compute" {
   }
 
   metadata = {
-    environment_root = var.environment_root
-    k3s_token = var.k3s_token
-    control_address = var.control_address
+    environment_root   = var.environment_root
```

Comment:
This seems pretty redundant. The idea of having `compute_init_enable` as a list matching the metadata was that it would be easy to construct this. Something like:

```
metadata = merge(
    {
        environment_root = var.environment_root
        ...
     },
    {for e in var.compute_init_enable: e => true})
)
```